### DISCUSSION TITLE
No wifi, no sign of panel life, no nothin

### DISCUSSION THREAD

**Discussion by Subarubrat** [thehighnibble/thehighnibble.github.io#131] [IMSAI8080esp replica]
Pertinent info:
September 2019 version 3.4.3 sat in box since purchase, just now got time to build it
Red LED lights on ESP32 immediately when the USB is plugged in
Wifi antenna is undamaged, not touching any nearby pins
No new wifi AP of any name appears, tested with multiple PCs and a phone
PC does not detect a new USB device and no new com port
Basic tests like lighting front panel LEDs with jumper works
All behavior is the same with the ESP module socketed or on its own
SD card works and has data on it

I got to the test/ testing1 steps and other than the red LED lighting up on the ESP32 there is no indication of life from the ESP32, I would jump to the conclusion of a DOA module but as I understand it the module was flashed before shipping, very odd it would take a flash and then go tango uniform. I wanted to dig deeper before asking for replacement on a four year old module. I populated and tested the LEDs on the panel in case that gave any indication of activity, but nothing. No response to the "power switch" on the panel and no LED activity. I don't have any prior experience with the ESP32 so maybe I am missing something, but I am out of ideas as to what.

**Comment by pfcushing** [comment]
So, by your "test/testing1 steps", do you really mean step 8 of the Step-by-step Assembly Guide or are you following some other assembly sequence?

Paul

**Comment by Subarubrat** [comment]
The official step by step guide. Step 8 is labeled as "test", "testing 1" has no number but is immediately after step 26.

**Comment by Subarubrat** [comment]
The official step by step guide. Step 8 is labeled as "test", "testing 1"
has no number but is immediately after step 26.

-Scott

**Comment by pfcushing** [comment]
Yup, sorry, I saw your reply but didn't get a chance to address it.  I checked my working Imsai8080 Replica and verified that the red led on the ESP comes on with power applied via the USB cable.  It does not flash or flicker showing activity, just solid red.

Did you plug the USB cable into a PC or just a 5V power module?

Paul Cushing

**Comment by Subarubrat** [comment]
I plugged it into two PCs (Linux) and no detection of any kind happened.
No wifi access point appeared, other than the LED there is no sign of
life. I have also powered it up in the 8080 with a USB power supply and
even connected to the PC just in case and nothing but the LED seems to
work.

  -Scott

**Reply by pfcushing** [reply]
i am a windows guy so i have no idea what indications you would get when plugging-in a new USB device.  In windows you get a notification that it is trying to install a driver for that USB device.  It was a couple of years ago but I think mine failed that step and that is why one might need to download a driver.  Since wi-fi isn't working either, I believe that you need the usb working so you can connect to it with something like Putty so you can see the messages that the ESP puts out when starting up, which was covered back at step 8.  Do you by chance also have a windows PC?

Paul Cushing

**Comment by Subarubrat** [comment]
In Linux they generally connect easily (I use similar modules for
emulating old school modems) and they also connect as a native TTY
terminal much easier. I ran over to my neighbor's house and tried it on
his Windows laptop, and yes there are windows drivers, but no
notifications of new devices, no new ports to connect to in putty to watch
boot messages. I did connect an ESP8266 module on the Windows machine and
it responded normally.

-Scott

**Reply by pfcushing** [reply]
Well, if you saw drivers install successfully on your neighbor's windows machine then I guess your initial belief of an ESP failure is right on.

Send a note to info@TheHighNibble.com and see what Dave's thoughts are.

Paul Cushing

**Comment by nbreeden2** [comment]
Is the cable you're using a 'charging only cable (has no data pins)? These are fairly common, it will power up the IMSAI but won't show up as a connected serial port on the PC.

**Comment by Subarubrat** [comment]
Multiple cables that worked with other devices were tried, same result.

**Comment by nbreeden2** [comment]
I'd guessed that would be the case. Some additional notes.
Without a working ESP32 the front panel won't work. The emulation does things like "Sense RESET is pushed and send the data to the LEDs drivers to light them in test mode". Without a working ESP32 the FP won't do anything. The same is true for the power switch, the emulation monitors the switch and again takes action to bring the panel to life (light some LEDs) when its clicked to on.

This may be your issue with Windows. On my WIN11 machine I had to manually install the drivers for the ESP32-Pico-Kit serial connection.
https://www.silabs.com/developers/usb-to-uart-bridge-vcp-drivers?tab=downloads
Windows update and automatic driver update didn't find them.

I have a video there that dives into manually flashing the firmware on the ESP32 that covers the driver install.
https://youtu.be/DbyHedQI1kA

**Comment by Subarubrat** [comment]
Tracking on the front panel. Although the ESP32 showed no signs of life other than the LED when connected to any computer through any cable I tried it in the unit in case it was up and running but having some sort of USB issue, but no sign of life there either. I can plug an ESP 8266, Lora, Pi, or any other device up to the PC and I get the expected result. I think I just have a DOA ESP32.

**Comment by nbreeden2** [comment]
Just to double check. You did solder the extra 6 pins to the ESP32 such that all of the ESP32 vias have pins? There are 3 pins per side that need to be added.
